### DISCUSSION TITLE
如何将我之前的文件找回来？（损失如何找回来）

### DISCUSSION THREAD

**Discussion by Leelagozero** [Yuukiy/JavSP#413] [General]
使用这个软件后，我自己的命名信息全部没有了，文件夹也全部都乱了。我花费几年时间整理出来的东西，你个软件给我毁了。我就想知道，如何才能复原，复原。

**Comment by Yuukiy** [comment]
帮你改正了个错别字。[前情提要](https://github.com/Yuukiy/JavSP/issues/415)

**Comment by newday-life** [comment]
后果自负
